_posts/en/newsletters/2024-10-25-newsletter.md

```diff
@@ -0,0 +1,174 @@
+  included in a block.  Currently, lightweight clients must download an
+  entire block to perform the same level of verification of a channel
+  announcement.
+
+  Mouton's post also briefly discusses allowing opt-in announcement of
+  existing simple taproot channels.  Due to the current lack of support
+  for announcements of non-P2WSH channels, all existing taproot channels
+  are [unannounced][topic unannounced channels].  A possible feature
+  that can be added to the proposal will allow nodes to signal to their
+  peers that they want to convert an unannounced channel to a public
+  channel.
+
+- **Draft BIP for sending silent payments with PSBTs:** Andrew Toth
+  [posted][toth sp-psbt] to the Bitcoin-Dev mailing list a draft BIP for
+  allowing wallets and signing devices to use [PSBTs][topic psbt] to
+  coordinate the creation of a [silent payment][topic silent payments].
+  This continues previous discussion about an earlier iteration of the
+  draft BIP, see Newsletters [#304][news304 sp] and [#308][news308 sp].
+  As mentioned in those earlier newsletters, a special requirement of
+  silent payments over most other PSBT-coordinated transactions is that
+  any change to a not-fully-signed transaction's inputs requires
+  revising the outputs.
+
+  The draft only addresses the most common expected situation where a
+  signer has access to the private keys for all inputs in a transaction.
+  For the less common situation of multiple signers, Toth writes that
+  "this will be specified in a following BIP".
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Duplicate blocks in blk*.dat files?]({{bse}}124368)
+  Pieter Wuille explains that, in addition to the current best chain of blocks,
+  the block data files can also include stale blocks or duplicate block data.
+
+- [How was the structure of pay-to-anchor decided?]({{bse}}124383)
+  Antoine Poinsot describes the structure of the [pay-to-anchor (P2A)][topic
+  ephemeral anchors] outputs included as part of Bitcoin Core 28.0's [policy
+  changes][bcc28 guide]. The [bech32m][topic bech32] encoded, 2-byte length, v1
+  witness program was chosen as a `bc1pfeessrawgf` vanity address.
+
+- [What are the benefits of decoy packets in BIP324?]({{bse}}124301)
+  Pieter Wuille outlines design decisions around the [inclusion of decoy
+  packets][bip324 decoy packets] in the [BIP324][] specification. The optional
+  decoy packets can be used to obfuscate traffic pattern recognition by
```

Comment:
Traffic patterns are "obfuscated", however traffic pattern *recognition* is more accurately "obstructed", "hindered", "hampered", "prevented", or "thwarted".